Procfile

```diff
@@ -0,0 +1 @@
+web: gunicorn list.wsgi
```

Comment:
%s/`list.wsgi`/`womenstemcommunities`.

Reply:
Didn't understand this one.

Reply:
In vim you `:%s` when replacing text. I will do it.

Reply:
In vim you :%s when replacing text. I will do.